electrolytes/solvation_shell_utils.py

```diff
@@ -0,0 +1,91 @@
+import copy
+import itertools
+import numpy as np
+from rmsd import kabsch_rmsd
+
+
+def filter_by_rmse(coords, n=20):
+    """
+    From a set of coordinates, determine the n most diverse, where "most diverse" means "most different, in terms of minimum RMSD.
+    We use the Kabsch Algorithm (https://en.wikipedia.org/wiki/Kabsch_algorithm) to align coordinates based on rotation/translation before computing the RMSD.
+    Note: The Max-Min Diversity Problem is in general NP-hard. This algorithm generates a candidate solution to MMDP for these coords
+    by assuming that the point 0 is actually in the MMDP set (which there's no reason a priori to assume). As a result, if we shuffled the order of coords, we would likely get a different result.
```

Comment:
Hm. I wonder if it's slightly better to select a random instance, rather than the first instance. Because presumably the first instance will occur (usually) early in the MD run, creating some bias.

Reply:
Agreed, will change that

Reply:
Please make sure you can set the random seed so that we can get repeatable results if we re-run the script.